demisto_sdk/commands/common/errors.py

```diff
@@ -874,6 +875,22 @@ def incident_type_integer_field(field):
     def incident_type_invalid_playbook_id_field():
         return 'The "playbookId" field is not valid - please enter a non-UUID playbook ID.'
 
+    @staticmethod
+    @error_code_decorator
+    def incident_type_auto_extract_fields_invalid(incident_fields):
+        return f"The following incident fields are not formatted correctly under " \
+               f"`fieldCliNameToExtractSettings`: {incident_fields}\n" \
+               f"Please format them in open of the following ways:\n" \
```

Comment:
something in the english here doesnt work for me..

Reply:
typo - fixing